packages/client/package.json

```diff
@@ -38,7 +38,7 @@
     "clean": "../../config/cli/clean-package.sh",
     "client:start": "ts-node bin/cli.ts",
     "client:start:dev1": "npm run client:start -- --discDns=false --discV4=false --bootnodes",
-    "client:start:dev2": "npm run client:start -- --discDns=false --discV4=false --transports=rlpx --port=30304 --dataDir=datadir-dev2",
+    "client:start:dev2": "npm run client:start -- --discDns=false --discV4=false --port=30304 --dataDir=datadir-dev2",
```

Comment:
Oh, THAT'S an impressive catch. 🙂 🙃

Reply:
Only because I'm pretty sure I originally added that.  :wink: